microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1685,9 +1761,12 @@
                   <div class="column report-text">
                     <h3>{{ _('Your Inner Zoo (aka What\'s in your sample?)') }}</h3>
                     <br>
-                    <p>
+                    <p class="site_specific_content site_specific_content-gut">
                     {{ _('This is a table of all of the different microbes we found in your sample along with their proportions.') }}
                     </p>
+                    <p class="site_specific_content site_specific_content-skin">
+                    {{ _('This is a table of all of the different bacteria we found in your sample along with their proportions.') }}
```

Comment:
so the skin info includes only bacteria while the gut info includes microbes from other domains of life?

Reply:
I brought this edit to Se Jin's attention and we're going to harmonize both gut and skin to say "bacteria and archaea" rather than "microbes" for gut and "bacteria" for skin.